.jenkins/pipelines/Intel/Linux/Jenkinsfile

```diff
@@ -13,7 +13,7 @@ AGENTS_LABELS = [
 // Jenkinsfiles and actual testing logic located <TBD>
 def TriggerIntelJob(String jobname) {
     stage("${jobname}") {
-        node("nonSGX") {
+        node("nonSGX-ubuntu-2004") {
```

Comment:
Since AGENTS_LABELS isn't used, any thoughts on either:
1. changing this node to AGENTS_LABELS["ubuntu-nonsgx"] or
2. removing AGENTS_LABELS

Alternatively we can just leave it as currently proposed.

Reply:
For consistency with other Jenkinsfile, I'm in favour of option 1

Reply:
Agreed!